package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "appium-adb",
-  "version": "3.0.0",
+  "version": "5.0.0",
```

Comment:
Please revert this

Reply:
also I'm not sure why `test/fixtures/selendroid-test-app.apk` is present in the PR

Reply:
Me either actually. It's in the repo, but likely it should probably be .gitignored?